nengo_bones/templates/.pre-commit-config.yaml.template

```diff
@@ -17,5 +17,12 @@ repos:
 -   repo: https://github.com/pycqa/isort
     rev: 5.6.4
     hooks:
-      - id: isort
-        files: \.py$
+    - id: isort
+      files: \.py$
+{% if py_typed %}
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.720
```

Comment:
Why is this 0.720, but in setup.py we require 0.800? (Also, wacky version numbers.)

Reply:
Ah just time passing from when I made the first version of this PR and the cleaned up one, haha. I'll update this.